rebasehelper/cli.py

```diff
@@ -151,6 +152,13 @@ def build_parser():
             help="continue previously interrupted rebase"
         )
         parser.add_argument(
+            "--color",
+            default='auto',
+            dest='color',
+            choices=['always', 'never', 'auto'],
+            help="use colors for CLI report"
```

Comment:
I'd use `colorize the output, defaults to %(default)s`.
Later we can add support for colorizing also other messages, not just the "CLI report".